calculatorUICOOP/ViewModels/MainPageViewModel.cs

```diff
@@ -1,24 +1,58 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
+﻿using System.ComponentModel;
+using System.Runtime.CompilerServices;
 
 namespace calculatorUICOOP.ViewModels
 {
-    public class MainPageViewModel
+    public class MainPageViewModel : INotifyPropertyChanged
     {
         #region Fields and Properties
```

Comment:
I like to keep fields and properties in 2 separate regions. They are not fuctionally the same.